doc/RFCs/messages-drafts-discussions-routes.md

```diff
@@ -0,0 +1,38 @@
+##### Routes pour les drafts :
+
+
+* `POST /messages` : création d'un draft
+    * ou `POST /drafts` ?
+* `PATCH /messages/{message_id}` : actualisation d'un draft
+* `DELETE /messages/{message_id}` : suppression d'un draft
+* `POST (ou PATCH?) /messages/{message_id}/send` : ordre d'envoi d'un draft (payload empty pour le moment, pourra ultérieurement contenir des instructions d'envoi par exemple : envoyer à une certaine date/heure)
+    * ou `POST (ou PATCH?) /messages/{message_id}/action`, avec un json en payload qui décrit l'action à accomplir : send pour le draft, read/unread, etc. pour les autres messages. Cette solution présente l'avantage d'avoin une seule route pour toutes les actions permises sur les messages. Exemple de payload possible : `{"action":"send"}`
+
+
+[Il faudra ultérieurement prévoir des routes pour la gestion des pièces jointes : upload, delete]
+
+##### Routes messages/discussions :
+
+* `GET /messages?` : une liste de messages selon les critères du filtre.  
+règles de filtrage envisageables :
+```
+    discussion_id=
+    caliopen-pi=
```

Comment:
if such filter exist, it's based on http headers
importance level range is more related to such client filtering